src/mutation.h

```diff
@@ -137,8 +137,13 @@ struct mutation_branch {
         float hearing_modifier = 1.0f;
         float noise_modifier = 1.0f;
         float scent_modifier = 1.0f;
+        cata::optional<int> scent_intensity = cata::nullopt;
+        cata::optional<int> scent_mask = cata::nullopt;
```

Comment:
Initialising the `cata::optional` that way is redundant. The default constructor already creates an empty object anyway.
```suggestion
        cata::optional<int> scent_mask;
```
Same for the other new member.